migrations/20171117161044_delete_networks_posts_dupes.js

```diff
@@ -0,0 +1,15 @@
+exports.up = knex => knex.raw(`
+  DELETE FROM networks_posts
+  WHERE id IN (
+    SELECT id
+    FROM (
+      SELECT
+        id,
+        ROW_NUMBER() OVER(PARTITION BY network_id, post_id ORDER BY id ) AS row_num
```

Comment:
🆒

Reply:
Indeed. This was like the bug fix that epitomizes what I THINK most bug fixes are going to be but often aren't 👍